fiber/middleware.go

```diff
@@ -0,0 +1,21 @@
+package fiber
+
+import (
+	"context"
+
+	"github.com/airbrake/gobrake/v5"
+
+	"github.com/gofiber/fiber/v2"
+)
+
+// New returns a function that satisfies fiber.Handler interface
+func New(notifier *gobrake.Notifier) fiber.Handler {
```

Comment:
Should we also have a test for this package?

Reply:
Great catch

Reply:
We may not require test coverage for middleware as we will be testing the method we are calling in the middleware.
I will check if we need to write a test for this middleware.